src/main/kotlin/no/nav/faktureringskomponenten/service/FakturaGenerator.kt

```diff
@@ -16,6 +19,15 @@ class FakturaGenerator(
     private val unleash: Unleash,
     @Value("\${faktura.forste-faktura-offsett-dager}") private val forsteFakturaOffsettMedDager: Long
 ) {
+    fun lagFaktura(
```

Comment:
Hvorfor introduseres denne metoden? Det er ikke tydelig for meg.

`lagFakturaerFor` inneholder logikk for å splitte fakturering per år, noe som er viktig å beholde

Reply:
Vi fikk det ikke til med eksisterende kode. Men slapp også opp for tid så gikk ganske fort på slutten i går.